pkg/admission/admission.go

```diff
@@ -347,3 +365,16 @@ func CreateWebhookName(handler WebhookHandler, suffix string) string {
 	}
 	return fmt.Sprintf("%s.%s.%s", webhookQualifier, subPath, suffix)
 }
+
+// bypassValidation users can bypass the webhook if they are the sudo account and system:masters group
+func bypassValidation(request *admissionv1.AdmissionRequest) bool {
```

Comment:
The description mentions we are targeting kinds `RoleTemplates` and `GlobalRoles`, however the bypass does not enforce that. Shall we add a check for that effect?

Reply:
Sorry, this bypass is meant to be used for all validation and mutation performed by the webhook. The first use case is for RoleTemplates and GloabalRoles. But as we migrate validation from Norman, we will also need to use this for other Resources. For example, Team2 just added some node driver validation that they need to bypass with their controller.